### PR TITLE
Added CAPA-RKE2 tests

### LINKED ISSUE #56: Automate CAPA+RKE2
Cluster template: https://github.com/rancher/cluster-api-provider-rke2/blob/main/samples/aws/cluster-template.yaml

### PR DESCRIPTION
**Pull request by cpinjani**
### What does this PR do?
- Adds CAPA-RKE2 test suite
- Fixes CAPD-RKE2 deletion issue

### Which issue(s) this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged):
Fixes #56 

### Checklist:
- [x] GitHub Actions: :green_circle: https://github.com/rancher/rancher-turtles-e2e/actions/runs/10681331189
(used this PR branch in fleet repo)

### CODE REVIEW

File: tests/assets/rancher-turtles-fleet-example/rke2_namespace_autoimport/clusters/cluster1.yaml [hunk 1]
```diff
@@ -176,3 +175,5 @@ kind: ConfigMap
 metadata:
   name: cluster1-lb-config
   namespace: default
+  annotations:
```
Comment (cpinjani) [inline tests/assets/rancher-turtles-fleet-example/rke2_namespace_autoimport/clusters/cluster1.yaml:178]: added as per https://github.com/rancher-sandbox/rancher-turtles-fleet-example/pull/11

File: tests/cypress/latest/e2e/unit_tests/capa_rke2_cluster.spec.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    // Click on imported CAPA cluster
+    cy.contains(clusterName).click();
+
+    // Install App
+    cy.installApp('Monitoring', 'cattle-monitoring');
+  })
+
+  qase(15,
+    it('Remove imported CAPA cluster from Rancher Manager', { retries: 1 }, () => {
+
```
Comment (thehejik) [inline tests/cypress/latest/e2e/unit_tests/capa_rke2_cluster.spec.ts:60]:
```suggestion

```

File: tests/cypress/latest/e2e/unit_tests/capa_rke2_cluster.spec.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      // kubectl get clusters.cluster.x-k8s.io
+      // This is checked by ensuring the cluster is not available in navigation menu
+      cy.contains(clusterName).should('not.exist');
+      cy.checkCAPIClusterProvisioned(clusterName);
+    })
+  );
+
+  qase(16,
+    it('Delete the CAPA cluster fleet repo', () => {
+
```
Comment (thehejik) [inline tests/cypress/latest/e2e/unit_tests/capa_rke2_cluster.spec.ts:73]:
```suggestion

```

**Review by thehejik — APPROVED** [review]
Thanks, LGTM

**Pull request merged**